<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVisitByQueueDialog</class>
 <widget class="QDialog" name="ReportVisitByQueueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Протокол обращений пациента по предварительной записи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>с</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBeginDateVisitBeforeRecordClient">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>по</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDateVisitBeforeRecordClient">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructureVisitBeforeRecordClient"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSpecialityVisitBeforeRecordClient"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPersonVisitBeforeRecordClient"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="chkNextVisitBeforeRecordClient">
     <property name="text">
      <string>Учитывать назначение следующей явки</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="chkNoVisitBeforeRecordClient">
     <property name="text">
      <string>Учитывать только не явившихся на прием</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QCheckBox" name="chkVisitOtherSpeciality">
     <property name="text">
      <string>Не учитывать явившихся к другому врачу данной специальности</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblSorting">
     <property name="text">
      <string>Сортировка</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QComboBox" name="cmbSorting">
     <item>
      <property name="text">
       <string>по дате</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по ФИО пациента</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по идентификатору пациента</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBeginDateVisitBeforeRecordClient</tabstop>
  <tabstop>edtEndDateVisitBeforeRecordClient</tabstop>
  <tabstop>cmbOrgStructureVisitBeforeRecordClient</tabstop>
  <tabstop>cmbSpecialityVisitBeforeRecordClient</tabstop>
  <tabstop>cmbPersonVisitBeforeRecordClient</tabstop>
  <tabstop>chkNextVisitBeforeRecordClient</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVisitByQueueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVisitByQueueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
